interpreters/wamr/Kconfig

```diff
@@ -83,6 +83,15 @@ config INTERPRETERS_WAMR_LIBC_WASI
 		not implemented. (Mainly because of lack of openat family of
 		the API in NuttX.)
 
+config INTERPRETERS_WAMR_LIBC_NUTTX
+	bool "Bypass to NuttX libc"
+	default n
```

Comment:
Please include --help with more information: 1) Why to bypass libc? 2) Scenarios where it could be useful, etc